internal/telemetry/client.go

```diff
@@ -225,14 +236,46 @@ func (c *Client) Stop() {
 		return
 	}
 	c.started = false
-	c.flushT.Stop()
 	c.heartbeatT.Stop()
 	// close request types have no body
 	r := c.newRequest(RequestTypeAppClosing)
 	c.scheduleSubmit(r)
 	c.flush()
 }
 
+// ProductEnabled enqueues an app-product-change event that signals a product has been turned on/off.
+// the caller can also specify additional configuration changes (e.g. profiler config info),
+// which will be sent via the app-client-configuration-change event
+func (c *Client) ProductEnabled(namespace Namespace, enabled bool, configuration []Configuration) {
+	c.mu.Lock()
+	defer c.mu.Unlock()
+	if !c.started {
+		c.log("attempted to send product change event, but telemetry client has not started")
+		return
+	}
+	productReq := c.newRequest(RequestTypeAppProductChange)
+	products := new(Products)
+	switch namespace {
```

Comment:
Why isn't tracing in here? 🤔

Reply:
Hmmm for some reason instrumentation telemetry API does not consider tracing as a `Product`

Reply:
The api seems to revolve around the tracer, while appsec and the profiler are considered external products added to the environment